vars/runTest.groovy

```diff
@@ -185,4 +185,5 @@ def call(Map config = [:]) {
 	    }
     }
 
+    return ['run_test': status]
```

Comment:
Doesn't the catchError() set `currentBuild.currentResult` to final status of this stage, negating the need to pass it back explicitly?

Reply:
This is mainly a simple example.   This can be a more complex Map with more data if needed.